assets/src/containers/FilesAccessed.js

```diff
@@ -127,8 +143,8 @@ function FilesAccessed (props) {
 
   useEffect(() => {
     // Fetch data once all the setting data is fetched
-    if (dataControllerLoad === 2) {
-      const dataURL = `/api/v1/courses/${courseId}/file_access_within_week?week_num_start=${weekRange[0]}&week_num_end=${weekRange[1]}&grade=${gradeRangeFilter}`
+    if (dataControllerLoad === 2 && resourceFilter != "") {
```

Comment:
`!=` should be changed to `!==`, to avoid [type coercion](https://stackoverflow.com/questions/19915688/what-exactly-is-type-coercion-in-javascript)